kinto_client/importer.py

```diff
@@ -0,0 +1,348 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import argparse
+import getpass
+import os
+from six import iteritems
+
+from kinto_client import Client
+from kinto_client.exceptions import KintoException
+from kinto_client.logging import logging, global_logger, command_log
+
+
+def is_same_record(fields, one, two):
+    for key in fields:
+        if one.get(key) != two.get(key):
+            return False
+    return True
+
+
+class KintoImporter(object):
+    # Default configure parser values
+    include_all_default_parameters = False
+    include_remote_server = None
+    include_authentication = None
+    include_files = None
+    include_verbosity = None
+
+    # Remote Permissions
+    bucket_permissions = None
+    collection_permissions = None
+
+    # Default values
+    default_host = "http://localhost:8888/v1"
+    default_bucket = None
+    default_collection = None
+    default_auth = None
+    default_files = None
+
+    # Sync flags
+    create = True
+    update = True
+    delete = True
```

Comment:
If these values are supposed to be changed during class override, they deserve detailed docstrings.